res/fm/rms/rms_run.py

```diff
@@ -164,7 +165,18 @@ def run(self):
             )
 
     def exec_rms(self, exec_env):
-        args = [self.config.wrapper] if self.config.wrapper is not None else []
+        # The rms exec environement needs to be injected betweed executing the
+        # wrapper and launching rms. PATH_PREFIX must be set in advance.
+        prefix_path = exec_env.pop("PATH_PREFIX", "")
+        env_args = ["env"]
+        for key, value in exec_env.items():
+            env_args.append(f"{key}={value}")
+        args = (
+            ["env", f"PATH_PREFIX={prefix_path}", self.config.wrapper] + env_args
```

Comment:
would this line have a duplication and is it done on purpose `"env"`

Reply:
so env should be called twice, once before the wrapper and once in between the wrapper and rms-executable

Reply:
The first one should just set the PATH_PREFIX

Reply:
understood